pii_data.py

```diff
@@ -8,7 +8,7 @@ class Pii(str):
     # https://regex101.com
     # https://www.w3schools.com/python/python_regex.asp
     def has_us_phone(self):
-        # Match a US phone number ddd-ddd-dddd ie 123-456-7890
+        # Match a US phone number ddd-ddd-dddd ie 123-456-7890s
```

Comment:
Was the s here intended or just a type-o?  If it was an error you can remove it and push your branch again

Reply:
it was a typo, i fixed it.